### ISSUE TITLE
Precision parameter for bytes

### ISSUE THREAD

**Issue by sergkr** [danrevah/ngx-pipes#54]
**I'm submitting a ...**  (check one with "x")
```
[ ] bug report => search github for a similar issue or PR before submitting
[x] feature request
```
Currently, there does not appear to be a way to limit the precision of the `bytes` pipe. `{{12345678 | bytes}}` outputs `12.345678 MB`.

Would be nice to be able to specify the maximum number of digits after the decimal. For example:

`{{12345678 | bytes: 2}}`

Which would round the output to `12.3 MB`.

**Comment by danrevah** [comment]
Good idea.

You are welcome to submit a pull-request. 
It's possible to use the `round` pipe in the `bytes` pipe instead of multiplying it's logic.

**Comment by danrevah** [comment]
It's Implemented in `v1.6.1`
